docs/index.md

```diff
@@ -47,22 +45,6 @@ To begin using the device, connect it to a USB - UART converter that operates on
 
 Visit the [Device Manager](device-manager.md) menu for more connection options.
 
-## Principle of Operation
```

Comment:
We are moving this specific information to somewhere else ?

Reply:
it is now at https://bluerobotics.com/learn/ping-sonar-technical-guide/#operating-principle